.github/workflows/main.yml

```diff
@@ -20,17 +20,17 @@ jobs:
 
       - uses: actions/setup-node@v1
         with:
-          node-version: '10.18.1'
+          node-version: '14.14.0'
```

Comment:
Move to latest version of node. I thought earlier this was causing errors in the builds, but it wasn't. Though, may as well move to a later version.